.eslintrc

```diff
@@ -2,9 +2,10 @@
   "extends": "@logux/eslint-config/esm",
   "overrides": [
     {
-      "files": "**/worker.ts",
+      "files": "**/**.js",
       "rules": {
-        "import/extensions": "off"
+        "import/extensions": "off",
```

Comment:
Why do we need these changes for code style?

Reply:
I didn't understand why we need this rule. Why I should move all my constant outside the of functions?

Reply:
1. The first rule is to add `.js` to the end of every `import` since it is requirements by ESM spec.
2. This second rule is to use `let` instead of `const`.

Reply:
Don’t worry, this is why review is useful